.eslintrc

```diff
@@ -1 +1,10 @@
-{ "extends": "scality" }
+{
```

Comment:
I wouldn't change eslint rules for the PR, not sure why JSX would need to be true either way.

Reply:
Yes thanks, with your changes it's all good! I am also based on your PR now.